functions/src/custom/event.ts

```diff
@@ -66,9 +67,17 @@ export const create_event = async (document: FirebaseFirestore.DocumentData): Pr
       public: public_event,
     };
 
+    const message: slack_message = {
+      form_name: "Event Check-in Generator",
+      name: first_name + " " + last_name,
+      email: email,
+      url: `https://app.acmutd.co/checkin/${path_name}`,
+    };
+
     await create_map(data);
-    add_callback(`https://app.acmutd.co/checkin/${path_name}`, await get_auth_token());
-    send_dynamic_template(email_options);
+    await add_callback(`https://app.acmutd.co/checkin/${path_name}`, await get_auth_token());
+    await send_dynamic_template(email_options);
+    await log_to_slack(message);
```

Comment:
What happens in the case where user enters an invalid `path_name` in the typeform? Currently there is no feedback that the event creation was not possible with given `path_name` besides the absence of a confirmation email and the sentry error. Typeform tells user to expect a confirmation email even in case where event creation was not possible leading user to believe event creation was successful.

Reply:
Currently there's no support for reporting errors via slack notification, full reliance on just sentry. The current setup basically duplicates the information sent to the user in the email & also sends it via slack (so that other officers can see the newly provisioned vanity links/check-in links/etc). 

We could potentially create a new issue to post specific errors/validation messages as well via the catch block. This PR basically sets up the tooling to add that type of support but implementing that is outside the scope for the current issue.